.pre-commit-config.yaml

```diff
@@ -9,6 +9,7 @@ repos:
   hooks:
     # Run the linter.
     - id: ruff
+      args: [ --fix ]
```

Comment:
I don't necessarily disagree this might be useful, but it should be in an separate PR.

Reply:
This was done in #2004!